base/iterators.jl

```diff
@@ -24,6 +24,26 @@ import .Base:
 
 export enumerate, zip, rest, countfrom, take, drop, takewhile, dropwhile, cycle, repeated, product, flatten, partition
 
+"""
+    Iterators.map(f, iterators...)
+
+Create a lazy mapping.  This is another syntax for writing
+`(f(args...) for args in zip(iterators...))`.
+
+!!! compat "Julia 1.5"
+    This function requires at least Julia 1.5.
+
```

Comment:
Maybe cross-reference with the docstring of `Base.Generator`

Reply:
I don't think `Base.Generator` is included in the documentation.  This is partially why I want to add `Iterators.map`.